dipy/reconst/tests/test_mapmri_odf.py

```diff
@@ -0,0 +1,69 @@
+import numpy as np
```

Comment:
The coverage for the mapmri file needs to go higher. Now it's at 79%. Here are the lines that are not tested
```
dipy.reconst.mapmri  79%   129, 157-158, 169-187, 285, 518-525, 654-666
```